plugins/module_utils/omevv_utils/omevv_firmware_utils.py

```diff
@@ -287,6 +294,14 @@ def delete_firmware_repository_profile(self, profile_id):
             "DELETE", PROFILE_URI + "/" + str(profile_id))
         return resp
 
+    def resync_repository_profiles_from_ump(self):
+        """
+        Resyncs the repository profiles from UMP.
+
+        """
+        resp = self.omevv.invoke_request("POST", RESYNC_UMP_URI, {})
+        return resp
```

Comment:
return resp.json_data, applies to other places too

Reply:
There's no json_data returned from this. Only success response is needed. That's why returning response directly.